aeneas/src/main/CLOptions.v3

```diff
@@ -142,6 +142,8 @@ component CLOptions {
 		"Enable reachability analysis and normalization before running the program.");
 	def MA			= sharedOpt.newBoolOption("ma", true,
 		"Generate mixed arrays for arrays of tuples, rather than tuples of arrays.");
+	def NR			= sharedOpt.newBoolOption("nr", true,
```

Comment:
We'll have to come up with better names for these flags, as this terseness was probably a bad idea in retrospect. Suggestions welcome.

Reply:
Perhaps CLOptions should support long names with short aliases for options? Then we won't have to sacrifice clarity for terseness here.